### PR TITLE
Removed emoji for Windows compatibility

### LINKED ISSUE #10: Support colors and emojis in windows terminal
**Colors**

Currently used library `aurora` is not supporting Windows command line out-of-the-box. Some help with the topic can be found here: https://github.com/logrusorgru/aurora#windows or we can switch the library completely. 

For now Windows Command Line users see ASCII escape codes instead of nicely written colors.

** Emojis**
Same happens with emojis, we should find a better way to print them so they are visible on Windows as well.

### PR DESCRIPTION
**Pull request by denny241**
Closes #10

### CODE REVIEW

File: internal/app/loophole/loophole.go [hunk 1]
```diff
@@ -106,23 +105,23 @@ func printWelcomeMessage() {
 
 func startLoading(loader *spinner.Spinner, message string) {
 	if el := log.Debug(); !el.Enabled() {
-		loader.Prefix = emoji.Sprintf("%s ", message)
+		loader.Prefix = fmt.Sprintf("%s ", message)
 		loader.Start()
 	} else {
-		fmt.Println(emoji.Sprint(message))
+		fmt.Println(fmt.Sprint(message))
```
Comment (Morishiri) [inline internal/app/loophole/loophole.go:111]: Just `fmt.Println` would be enough

File: internal/app/loophole/loophole.go [hunk 2]
```diff
@@ -290,8 +289,8 @@ func generateListener(config lm.Config, publicKeyAuthMethod *ssh.AuthMethod, pub
 	fmt.Fprint(colorableOutput, aurora.Green(fmt.Sprintf("%s:%d", config.Host, config.Port)))
 	fmt.Println()
 	fmt.Println()
-	fmt.Fprint(colorableOutput, emoji.Sprint(fmt.Sprintf(":nerd: %s", aurora.Italic("TLS Certificate will be obtained with first request to the above address, therefore first execution may be slower\n"))))
-	fmt.Fprint(colorableOutput, emoji.Sprint(":newspaper: Logs:\n"))
+	fmt.Fprint(colorableOutput, fmt.Sprint(fmt.Sprintf("%s", aurora.Italic("TLS Certificate will be obtained with first request to the above address, therefore first execution may be slower\n"))))
```
Comment (Morishiri) [inline internal/app/loophole/loophole.go:292]: `fmt.Sprint(fmt.Sprintf` you have it twice

Comment (denny241) [reply internal/app/loophole/loophole.go:292]: I forgot that there were nested calls, I thought it would be fine to replace all the "emoji." with "fmt."

**Comment by 0x7f** [comment]
😄 
image

**Review by Morishiri — APPROVED** [review]

**Pull request merged**